From b84f1adc6985841923980d599488974d2b4cd573 Mon Sep 17 00:00:00 2001
From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Thu, 20 Dec 2018 19:49:41 +0100
Subject: [PATCH] modules: update Gluon packages

f49aef69e8e4 autoupdater: consider end of string smaller than all characters except for '~'
7348db07e2db autoupdater: fix regression in version compare
---
 modules | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/modules b/modules
index 266d9e2..4dc26e0 100644
--- a/modules
+++ b/modules
@@ -9,7 +9,8 @@ PACKAGES_OPENWRT_BRANCH=lede-17.01
 PACKAGES_OPENWRT_COMMIT=40da7ecf21ffe1f3523ffa430c406e1db58ce3d4
 
 PACKAGES_GLUON_REPO=https://github.com/freifunk-gluon/packages.git
-PACKAGES_GLUON_COMMIT=be2c35785994e443d895225c7240474a46f64f5e
+PACKAGES_GLUON_BRANCH=v2018.1.x
+PACKAGES_GLUON_COMMIT=f49aef69e8e454f236215fdea19f0ef5864e9acc
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt-routing/packages.git
 PACKAGES_ROUTING_BRANCH=openwrt-18.06
-- 
2.5.0

